src/backend/main/py.main/caliopen_main/message/store/message.py

```diff
@@ -47,3 +49,10 @@ class Message(BaseModel, IndexedModelMixin):
     tags = columns.List(columns.Text(), db_field="tagnames")
     protocol = columns.Text()
     user_identities = columns.List(columns.UUID)
+
+
+class MessageExternalRefLookup(BaseUserCore):
```

Comment:
it's not normal to see a core class in store namespace.

Reply:
changed with commit #dbbf0b3d